library/src/main/java/com/novoda/downloadmanager/Migrator.java

```diff
@@ -2,7 +2,7 @@
 
 import android.support.annotation.WorkerThread;
 
-public interface Migrator {
+interface Migrator {
```

Comment:
I'm under the impression that this should be an internal concept, clients should interact through the binder only.